test/file1-1.tsx

```diff
@@ -0,0 +1,129 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import { Collapse } from '@xsite-ui/common.collapse';
+import { Image } from '@xsite-ui/common.image';
+import { CtaButton } from '@xsite-ui/fintech.cta-button';
+import { PoweredByBankrate } from '@xsite-ui/fintech.icons/powered-by-bankrate';
+import { ProductIdentifier } from '@xsite-ui/fintech.product-identifier';
+import { OfferProduct, ProductTypes } from '@xsite-ui/fintech.shared-types';
+import { getMoneyString } from '@xsite-ui/fintech.utils';
+import React, { useState } from 'react';
+
+import { CollapseContent } from './collapse-content';
+import { COLLAPSE_BUTTON_TEXT } from './constants';
+import {
```

Comment:
'ArrowIcon', 'CollapseButton', 'CollapseButtonText', 'ctaButton', 'CtaWrapper', 'DataColumn', 'DataValue', 'identifier', 'poweredBy', 'ProductDataColumnContainer', 'ProductLogoContainer', 'productLogoImage', and 'ProductWrapper' are imported but not used in the code.